readthedocs/search/api.py

```diff
@@ -192,6 +194,12 @@ class PageSearchAPIView(GenericAPIView):
     - project
     - version
 
+    Optional params:
+
+    - new-api (true/false): make use of the new stable API.
```

Comment:
I wonder wonder if we should version this (eg. `version=2`). I think that's really what we're doing is versioning the response. I do wonder about just creating a new endpoint that uses the new serializer tho, and not doing this if it isn't too much complexity.

Reply:
yeah, having another endpoint shouldn't bee that complicated